sdv/constraints/tabular.py

```diff
@@ -892,19 +892,11 @@ def is_valid(self, table_data):
         middle = table_data[self.middle_column_name]
         high = table_data[self.high_column_name]
 
-        satisfy_low_bound = np.logical_or(
-            self._operator(low, middle),
-            np.isnan(low),
-        )
-        satisfy_high_bound = np.logical_or(
-            self._operator(middle, high),
-            np.isnan(high),
-        )
+        low_lt_middle = self._operator(low, middle) | np.isnan(low) | np.isnan(middle)
```

Comment:
I think one other small optimization we can make is storing the computation of nans in variables so we only run `np.isnan(column)` once for each column. So like
```python
low_is_nan = np.isnan(low)
middle_is_nan = np.isnan(middle)
low_lt_middle = self._operator(low, middle) | low_is_nan | middle_is_nan
...
```